include/rmm/mr/device_memory_resource.hpp

```diff
@@ -107,6 +108,17 @@ class device_memory_resource {
    *---------------------------------------------------------------------------**/
   virtual bool supports_streams() const noexcept = 0;
 
+  /**---------------------------------------------------------------------------*
+   * @brief Queries the amount of free and total memory for the resource.
+   *
+   * @param stream the stream whose memory manager we want to retrieve
```

Comment:
Let's make all `@param stream` comments consistent across all the MRs. Currently they are all written differently. 

(I actually find cnmem's putting mem_info on a stream not that useful -- I think usually users will want to know the *total* free and allocated memory, not associated with a specific stream).